src/app/containers/EpisodeList/Image.jsx

```diff
@@ -47,6 +47,9 @@ const PlayWrapper = withEpisodeContext(styled.div`
     height: 0.6rem;
     width: 0.7rem;
     color: ${C_WHITE};
+    @media screen and (forced-colors: active) {
+      fill: canvasText;
```

Comment:
Think this should be the link colour as it's wrapped in a link element in the output?